docs/camino-messenger/introduction.md

```diff
@@ -216,7 +216,39 @@ In the Camino Messenger Protocol, there is no static data inclusion in any of th
+The `v1alpha` version first release was done on 18 January 2023. Feedback and reviews with Camino Network Partners are leading to improvements of Message Types. A partner working with a specific Message Type to implement a connection to their system discovers errors or finds a concept is missing in the Camino MessageType. Conclusively one Message Type can be mature and and have only non-breaking improvements, while another Message Type is actively being developed. While "Accommodation" might be in a productive `v1`, we might have "Car Rental" in a `v2beta`. As all message formats are released together, we simply have numeric releases that contain updates on one or more messages from one or more feature branches and PRs.
+
+### Bot version (Synced with the protocol-release)
+
+Every time a new version is released of any of the message types a new version of the bot is released. The latest version of the Camino Messenger bot can be downloaded from our [github](https://github.com/chain4travel/camino-messenger-bot). The main bot version is the same as the latest protocol release. Technical releases for the bot can result to a minor version increase (for example bot release 6.0.0 and 6.1.0 which both support latest protocol version 6, but bot release 6.1.0 has a technical enhancement without an update to the protocol).
+
+### Protocol release
+
+A protocol release consists of the latest version of each message type. We call them _"service versions"_. The bot will always support the latest service version and the previous service version. The version before the previous version will still be in the SDK on buf.build, so that in the transition phase after the release of a new service version, it can still be used, but with "obsolete" warnings.
+
+Example: Partners A and B can still be trading on protocol version 4 (for example using Bot release 4.0.0), while partners C and D are trading on protocol version 4 (using the same bot release 4.0.0) and partners E and F are already trading on protocol version 5 of that message type using the most recent bot release 5.0.0. Upon release of the new Bot release 6.0.0, partners trading on version 4 of the message type will receive an "obsolete" warning with each message.
+
+### Service versions
+
+The version of each Message Type, _"service versions"_, are specified in the package name stored on [buf.build](https://buf.build/chain4travel/camino-messenger-protocol/docs) . The package name and conclusively the version is included in the gRPC header. To define the supported capabilities for the partner configuration, we can use this package name as a whole to refer to a service as a capability. For example: cmp.services.accommodation.v1alpha.AccommodationSearchService.
```

Comment:
> The package name and conclusively the version is included in the gRPC header.

I don't see that in the bot's metadata implementation. Where does this come from?